docs/preview/styles/css/style.css

```
@@ -10778,4 +10802,13 @@ textarea.form-control-lg {
   scrollbar-width: none; /* Firefox */
```

Ja u siebie mam taki zapis: 


```suggestion
::-webkit-scrollbar {
  display: none;
  -ms-overflow-style: none; /* IE and Edge */
  scrollbar-width: none; /* Firefox */
}